### ISSUE TITLE
Setting of self.sequential in HessianQuantCalibrator

### ISSUE THREAD

**Issue by yhwangs** [hahnyuan/PTQ4ViT#23]
In the `batching_quant_calib()` function of the `HessianQuantCalibrator` class in `quant_calib.py`, 
if `self.sequential` is set to `False` (the default value in the provided code), the quantization interval is not reflected in the calibration step of the next module after calibrating the current module. 
In this case, when calculating loss = KL_div(y, y_hat), instead of computing it between the output y of the full-precision (FP) model and the output y_hat of the quantized model, it would be calculated between two outputs of the FP model (i.e., KL_div(y, y)).

So, should `self.sequential` be set to `True`? 
However, when setting `self.sequential=True`, the accuracy is reported to be very low.
Could you please clarify this issue and provide guidance on the appropriate setting for `self.sequential`?
